src/platform/graphics/webgpu/webgpu-graphics-device.js

```diff
@@ -174,9 +174,17 @@ class WebgpuGraphicsDevice extends GraphicsDevice {
         // temporary message to confirm Webgpu is being used
         Debug.log("WebgpuGraphicsDevice initialization ..");
 
+        // build a full URL from a relative path
+        const buildUrl = (relativePath) => {
+            const url = new URL(window.location.href);
```

Comment:
I'm not sure we should have a `window` reference which would break in node. Can we just update the `createGraphicsOption` to include the full path, and assume any path passed in is correct?

Reply:
I think it's fine as long it's not e.g. in a variable declaration in global scope, or do you run into this code and trigger a reference error in node?

(before it was a simple `<script src=...=>` aswell and if you wanted to use that you would have already required the jsdom shim lib in node to have access to `window` etc. via node)

Reply:
Yeah I just wonder if we just require the path to be correct when it's passed in, rather than making the engine make figure it out. The createGraphics options just has the full url. Not sure, might be missing something obvious

Reply:
> correct when it's passed in

I agree with this. At the end, the application itself passes this path in, they should be able to build a correct path easier than the engine.